packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -65,7 +66,7 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return await this.provider.wasm.contractQuery(address, input, params)
   }
 
-  signAndSend = async (messages: MsgExecuteContract[]): Promise<TransactionResponse> => {
+  signAndSend = async (messages: MsgExecuteContract[] | MsgSend[]): Promise<TransactionResponse> => {
```

Comment:
Consider using a more general `Msg[]` type [as declared in the SDK](https://github.com/terra-money/terra.js/blob/fcfc3b0085546410a08cfa0207bd96227ad11762/src/core/Msg.ts#L83).